### PR TITLE
Fix TODO on Wasm object ownership

### PR DESCRIPTION
**Pull request by daviddrysdale**
Multiple choice PR:
 - [ ] Take just the first commit, which documents current ownership setup.
 - [ ] Take the second commit (squashed), which drops the extra reference.
 - [ ] Take the third commit (squashed), which uses an accessor to wrap `GetLastModule`

### CODE REVIEW

**Comment by daviddrysdale** [comment]
Ah, but which version do we want?  The first commit or the second?

**Comment by anghelcovici** [comment]
Ha, I've looked at the review and it gave me the second commit, didn't read your comment :(
I don't have strong feelings about this, @tiziano88 - any preference?

File: oak/server/wasm_node.cc [hunk 1]
```diff
@@ -234,7 +232,8 @@ void WasmNode::Run() {
 
   LOG(INFO) << "module execution thread: run oak_main";
   wabt::interp::TypedValues args;
-  wabt::interp::ExecResult exec_result = executor.RunExportByName(module_, "oak_main", args);
+  wabt::interp::ExecResult exec_result =
+      executor.RunExportByName(env_.GetLastModule(), "oak_main", args);
```
Comment (tiziano88) [inline oak/server/wasm_node.cc]: How about adding a method `Module()` defined as `env_.GetLastModule()` for clarity?

Comment (daviddrysdale) [reply oak/server/wasm_node.cc]: That gets my vote.

**Pull request merged**